pkg/render/manifests.go

```diff
@@ -234,6 +238,17 @@ func (c *clusterManifestContext) roksMetrics() {
 	)
 }
 
+func (c *clusterManifestContext) apiserverNetworkProxy() {
+	c.addManifestFiles(
+		"konnectivity/konnectivity-server-services.yaml",
+		"konnectivity/konnectivity-tugboat-agent-deployment.yaml",
+		"openshift-apiserver/konnectivity-proxy-configmap.yaml",
```

Comment:
Missing `openshift-apiserver/konnectivity-proxy-secret.yaml` and `konnectivity/konnectivity-server-certs.yaml` here.

Reply:
Secrets are handled in openshift-master's [cluster-secrets](https://github.ibm.com/alchemy-containers/armada-openshift-master/tree/master/roles/cluster-secrets) role, so they don't need to be included in this change.